interfaces/vrep/robots/LBR4pVrepRobot.m

```diff
@@ -8,7 +8,7 @@
 %           >> vi.connect('127.0.0.1',19997);
 %           >> vrep_robot = LBR4pVrepRobot("LBR4p", vi);
 %           >> vi.start_simulation();
-%           >> robot.get_q_from_vrep();
+%           >> robot.get_configuration_space_positions();
```

Comment:
Please change the methods' names according to my comments on the other files. The legacy names should be deprecated.

Reply:
Updated in [ecab5b5](https://github.com/dqrobotics/matlab/pull/108/commits/ecab5b5d6d0eacbe3a37633a3e48c4f877974075).